site/docs/sql/02-job-management.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Job Management
+description: Commands to manage SQL jobs
+---
+
+## CREATE/DROP/ALTER JOB
+
+These statements create a potentially long-running Jet job that is not
+tied to the client session.
+
+When you submit a standard INSERT query, its lifecycle is tied to the
+client session: if the client disconnects, the query is cancelled, even
+if it is supposed deliver results somewhere else (not back to the
+client).
+
+If you want to submit a statement that is independent from the client
+session, use the `CREATE JOB` statement. Such a statement will complete
+quickly and let the job running in the cluster. You can also configure
+it to be fault-tolerant.
+
+### CREATE JOB Synopsis
+
+```sql
+CREATE JOB [IF NOT EXISTS] job_name
+[ OPTIONS ( option_name 'option_value' [, ...] ) ]
+AS query_spec
+```
+
+- `job_name`: a unique name identifying the job.
+
+- `query_spec`: the query the job will run. Currently we support `INSERT
+  INTO` and `SINK INTO` queries. `SELECT` is not supported by design
+  because it returns the results to the client.
+
+- `option_name`, `option_value`: the job configuration options. The list
+  of options matches the methods in the `JobConfig` class.
```

Comment:
We should be more certain here because it seems that some of them are supported (for example `processingGuarantee`) but some of them not (for example `name`). I think it's also ambiguous - it's not clear whether name should be `processingGuarantee` or `setProcessingGuarantee` (it's can be understand from example but I think it should be mentioned here clearly).

Reply:
will create a new issue about this.